[2024-11-29T18:06:33.897+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-29T18:06:33.921+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo.print_the_hi scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2024-11-29T18:06:33.935+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo.print_the_hi scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2024-11-29T18:06:33.936+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-29T18:06:33.949+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): print_the_hi> on 2023-02-06 00:00:00+00:00
[2024-11-29T18:06:33.957+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'demo', 'print_the_hi', 'scheduled__2023-02-06T00:00:00+00:00', '--job-id', '748', '--raw', '--subdir', 'DAGS_FOLDER/zAGI/hello.py', '--cfg-path', '/tmp/tmpiphzls47']
[2024-11-29T18:06:33.959+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2043) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-29T18:06:33.960+0000] {standard_task_runner.py:105} INFO - Job 748: Subtask print_the_hi
[2024-11-29T18:06:33.960+0000] {standard_task_runner.py:72} INFO - Started process 2053 to run task
[2024-11-29T18:06:34.025+0000] {task_command.py:467} INFO - Running <TaskInstance: demo.print_the_hi scheduled__2023-02-06T00:00:00+00:00 [running]> on host 080dcf124ece
[2024-11-29T18:06:34.098+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo' AIRFLOW_CTX_TASK_ID='print_the_hi' AIRFLOW_CTX_EXECUTION_DATE='2023-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-06T00:00:00+00:00'
[2024-11-29T18:06:34.099+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-29T18:06:34.102+0000] {logging_mixin.py:190} INFO - hello world
[2024-11-29T18:06:34.102+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-29T18:06:34.112+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-29T18:06:34.113+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=demo, task_id=print_the_hi, run_id=scheduled__2023-02-06T00:00:00+00:00, execution_date=20230206T000000, start_date=20241129T180633, end_date=20241129T180634
[2024-11-29T18:06:34.146+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-29T18:06:34.157+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-29T18:06:34.159+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
